### PR TITLE
Logger

### PR DESCRIPTION
**Pull request by roytman**
moved from `fmt.Printf` to `klogv2`

### CODE REVIEW

File: pkg/json/encode.go [hunk 1]
```diff
@@ -40,3 +40,27 @@ func (s Set) MarshalJSON() ([]byte, error) {
 	buf.WriteString(`]]`)
 	return buf.Bytes(), nil
 }
+
+func (cmr *CondMonitorParameters) UnmarshalJSON(p []byte) error {
```
Comment (hunchback) [inline pkg/json/encode.go:44]: how is this related to the klog change ? better to separate as it makes reviewing and merging 10x  faster

Comment (roytman) [reply pkg/json/encode.go:44]: It's from the previous PR, this PR is based on the first one

**Comment by hunchback** [comment]
@roytman can you please split this into two separate PRs:
1. moving to standard logger
2. initial implementation of monitor

by connecting 2 unrelated issues together this makes all handling (review, testing, etc) 4 times more time consuming. Now if one is based on the other then the practice I would recommend:
1. logger PR - includes only the logger commit
2. monitor PR - includes both logger + monitor commits and has a commend to say it depends on the 1st PR and should only be processed after the first PR has been merged (and then via rebase the logger commit will go away)

File: pkg/ovsdb/ovsdb.go [hunk 2]
```diff
@@ -377,7 +374,7 @@ func (s *ServOVSDB) getMonitoredData(dataBase string, conditions map[string][]ov
 			}
 			returnData[tableName] = d1
 		} else {
-			// TODO we have to finalize the data store schema for NB and SB databases.
+			// TODO
```
Comment (hunchback) [inline pkg/ovsdb/ovsdb.go:377]: revert

File: pkg/ovsdb/ovsdb.go [hunk 3]
```diff
@@ -353,11 +350,11 @@ func (s *ServOVSDB) getMonitoredData(dataBase string, conditions map[string][]ov
 		}
 		if dataBase == "_Server" && tableName == "Database" {
 			if len(mcrs) > 1 {
-				fmt.Printf("!!!!! MCR is not a singe %v\n", mcrs)
+				klog.Warningf("MCR is not a singe %v", mcrs)
 			}
 			d1 := map[string]ovsjson.RowUpdate2{}
 			for _, v := range resp.Kvs {
-				fmt.Printf("key = %v \n", string(v.Key))
+				//	keys := strings.Split(string(v.Key), "/")
```
Comment (hunchback) [inline pkg/ovsdb/ovsdb.go:357]: please remove this line

**Review by hunchback — APPROVED** [review]
with minor comments

File: pkg/ovsdb/ovsdb.go (hunk 3 above)
Comment (roytman) [reply pkg/ovsdb/ovsdb.go]: done
